tests/test_spec.py

```diff
@@ -100,3 +111,12 @@ def test_two_endpoints_with_the_same_path():
     http_methods = list(spec['paths']['/lone'].keys())
     http_methods.sort()
     assert http_methods == ['get', 'post']
+
+
+def test_valid_openapi_spec():
+    app = create_app()
+    api.register(app)
+    spec = api.spec
+    print(json.dumps(spec))
+    test = validate_v3_spec(spec)
+    breakpoint()
```

Comment:
can remove the debug print/breakpoint

Be good to flesh out these tests to cover more complex models but nice one 👍

Reply:
Hah yes! thank you